api/src/opentrons/protocol_api/core/engine/well.py

```diff
@@ -98,8 +115,22 @@ def get_bottom(self, z_offset: float) -> Point:
     # Make this change carefully with respect to the robot-server because
     # `WellOrigin` is a public enum that may be persisted
     def get_center(self) -> Point:
-        """Get the coordinate of the well's center, with an z-offset."""
+        """Get the coordinate of the well's center, with a z-offset."""
```

Comment:
`get_center` doesn't take a z-offset, this was a copy paste error

```suggestion
        """Get the coordinate of the well's center."""
```